app/src/main/java/com/sun/spoonacular/data/source/remote/api/ApiService.kt

```diff
@@ -1,15 +1,23 @@
 package com.sun.spoonacular.data.source.remote.api
 
+import com.sun.spoonacular.data.model.RecipeDetail
 import com.sun.spoonacular.data.model.RecipeResponse
-import com.sun.spoonacular.utils.Constant
 import retrofit2.Response
 import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
 
 interface ApiService {
 
-    @GET(Constant.RANDOM_5)
-    suspend fun getRecipe() : Response<RecipeResponse>
+    @GET("random")
+    suspend fun getRecipe(@Query("number") number: Int = 5): Response<RecipeResponse>
 
-    @GET(Constant.RANDOM_10)
-    suspend fun getRecipes() : Response<RecipeResponse>
+    @GET("random")
+    suspend fun getRecipes(@Query("number") number: Int = 10): Response<RecipeResponse>
```

Comment:
2 hàm này khác gì nhau vậy em?

Reply:
> 2 hàm này khác gì nhau vậy em?

Dạ giống nhau a, nhưng mỗi thằng sẽ đảm nhiệm get data cho mỗi chỗ, hôm trước em get chung rồi dùng for loop để chia data ra nhưng a Tuán nói không được làm như vậy